src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,34 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={post.user} />
+      </p>
+    </div>
+
+    <p className="PostInfo__body">
+      est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae ea
+      dolores neque fugiat blanditiis voluptate porro vel nihil molestiae ut
+      reiciendis qui aperiam non debitis possimus qui neque nisi nulla
+    </p>
+
+    {post.comments.length !== 0 ? (
```

Comment:
```suggestion
    {post.comments.length ? (
```
unnecessary check, all numbers in JS give true, and 0 gives false